x-pack/plugins/security_solution/public/entity_analytics/components/risk_score_page_styles.tsx

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import styled from '@emotion/styled';
+import { css } from '@emotion/react';
+
+import type { SerializedStyles } from '@emotion/react';
+import type { EuiThemeComputed } from '@elastic/eui';
+interface EntityAnalyticsRiskScorePageStyles {
+  ButtonStyle: SerializedStyles;
+  VerticalSeparator: ReturnType<typeof styled.div>;
+  ToggleStyle: SerializedStyles;
+  DatePickerStyle: SerializedStyles;
+}
+
+export const getEntityAnalyticsRiskScorePageStyles = (
+  euiTheme: EuiThemeComputed
+): EntityAnalyticsRiskScorePageStyles => ({
+  ButtonStyle: css`
+    height: ${euiTheme.size.base};
+    padding: 0 ${euiTheme.size.s};
+    font-size: ${euiTheme.size.m};
+  `,
+
+  VerticalSeparator: styled.div`
+    :before {
+      content: '';
+      height: ${euiTheme.size.l};
+      border-left: ${euiTheme.border.width.thin} solid ${euiTheme.colors.lightShade};
+    }
+  `,
+
+  ToggleStyle: css`
+    height: ${euiTheme.size.l};
+    width: ${euiTheme.size.xl};
+  `,
+
+  DatePickerStyle: css`
+    flex-grow: 1;
+    max-width: ${euiTheme.size.xl};
+    font-size: ${euiTheme.size.l};
+    input {
+      height: ${euiTheme.size.l};
+      padding: ${euiTheme.size.s};
+      border: ${euiTheme.border.width.thin} solid ${euiTheme.colors.lightShade};
+      border-radius: ${euiTheme.border.radius.small};
+    }
+    @media (max-width: 100vw) {
+      max-width: 100%;
+    }
```

Comment:
I won't block the PR, but FYI:

Inside Kiban, we try to use the EuiLibrary as much as possible. Writing custom CSS is a fallback that should be used sparingly to prevent the pages inside Kibana from looking completely different.

For example:
* Instead of `flex-grow: 1,` we use https://eui.elastic.co/#/layout/flex
* Instead of `font-size: ${euiTheme.size.l};` we use https://eui.elastic.co/#/display/text
* Instead of `@media (mac-width)`, we use https://eui.elastic.co/#/theming/breakpoints/values (some components like `EuiFlexGroup` and `EuiFlexItem` have `responsive:boolean` that makes this logic simpler and that is usually enough)

We also avoid overwriting EuiLibrary CSS as much as possible because it easily breaks between EuiLibrary versions. 

If the designer creates a mockup different from what we have on EuiLibrary, we try to push back until there is a reasonable justification for the new look. This prevents Kibana's pages from looking completely different.

When we follow these principles and try to work with the library instead of working around it, writing custom becomes almost unnecessary.